core/src/main/java/io/questdb/log/LogRollingFileWriter.java

```diff
@@ -416,8 +417,12 @@ private void removeExcessiveLogs(long filePointer, int type) {
                 int startOffset = logFileNameSink.length();
                 logFileNameSink.put(logFileName);
                 int endOffset = logFileNameSink.length();
+                // It will be sorted as 128 bits hence
+                // set 2 longs for an entry, [packed_offsets, last_modification_ts]
+                // and it will sort it by last_modification_ts first and then by packed_offsets
+                long packed_offsets = Numbers.encodeLowHighInts(startOffset, endOffset);
```

Comment:
nit: variable names in Java should be in camelCase
```suggestion
                long packedOffsets = Numbers.encodeLowHighInts(startOffset, endOffset);
```